### PR TITLE
adding appcenter lab

### PR DESCRIPTION
**Pull request by sriramdasbalaji**

### CODE REVIEW

File: labs/vstsextend/appcenter/README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+layout: page
+sidebar: vsts2
+permalink: /labs/vstsextend/appcenter/
+folder: /labs/vstsextend/appcenter/
+---
+<div class="rw-ui-container"></div>
+
+## Overview
+
+[Visual Studio App Center](https://appcenter.ms/) brings together multiple services commonly used by mobile developers into an integrated cloud solution. Developers use App Center to Build, Test, and Distribute applications. Once the app's deployed, developers monitor the status and usage of the app using the Analytics and Diagnostics services, and engage with users using the Push service.
```
Comment (hsachinraj) [inline labs/vstsextend/appcenter/README.md]: Visual Studio App Center is an integrated mobile development lifecycle solution for iOS, Android, Windows and macOS apps. AppCenter brings together multiple services commonly used by mobile developers, including build, test, distribute, monitor, diagnose , etc., into one single cloud solution.

File: labs/vstsextend/appcenter/README.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+In this lab, we are going to Automate the **Build-Test-Distribute** process for **Tailwind Traders** mobile applications using **App Center + GitHub**. Tailwind Traders is a fictitious retail company showcasing the future of intelligent application experiences. These reference apps are powered by the Azure cloud, built with best-in-class tools, and made smarter through data and AI. The source repository used  in this lab is  imported from the below location
```
Comment (hsachinraj) [inline labs/vstsextend/appcenter/README.md]: 1. automate must be in lower case

1. mobile application using **App Center**.  The code for the mobile application is on GitHub at  [TailwindTraders-Mobile](https://github.com/Microsoft/TailwindTraders-Mobile)

File: labs/vstsextend/appcenter/README.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+  * Personal Access Token (PAT) for GitHub with *repo* scope
+
+## Task 1: Set up the project in **App Center** using **App Center Demo Generator**
+
+1. Open [App Center Demo Generator](https://appcenterdemogenerator.azurewebsites.net/) site on your browser. Click the links in the **Prerequisites** section to generate the required API tokens mentioned above.
+   
+    ![](images/homepage.png)
+
+1. Enter the valid **App Center API Token** in the **App Center API Token** text field and click the **Verify & Continue** button.
```
Comment (hsachinraj) [inline labs/vstsextend/appcenter/README.md]: 1. Enter a valid **App Center API Token**

File: labs/vstsextend/appcenter/README.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+   
+   ![](images/appcentertoken.png)
+
+1. In the resulting page, choose **Tailwind Traders** template from a number of templates by clicking the **...** *ellipsis* button next to **Selected App Template** field. **Tailwind Traders** is the default selected template.
```
Comment (hsachinraj) [inline labs/vstsextend/appcenter/README.md]: **Tailwind Traders** is the default selected template. But the App Center Demo Generator has several other templates which can be chosen by selecting the **ellispis (...)** button next to the **Selected App Template** field

File: labs/vstsextend/appcenter/README.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+     
+    ![](images/choosetemplate.png) 
+    ![](images/selecttailwind.png)
+
+1. Once the template is selected, a name for the application is auto-generated. Enter the valid GitHub token that was created previously. If you haven't created the token yet, click the **Generate a token** to generate a token with **repo** access. Click the **Create App** button.
```
Comment (hsachinraj) [inline labs/vstsextend/appcenter/README.md]: Enter a valid GitHub token

File: labs/vstsextend/appcenter/README.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+
+1. In  App Center portal **TailwindTraders_xxxxx** which was created is known as **Organization**. Organizations are used to manage multiple apps. And under this Organization, we have two apps. One is for Android and other one is iOS. For more information on Creating and managing organizations click [here](https://docs.microsoft.com/en-us/appcenter/dashboard/creating-and-managing-organizations).
+     
+     ![](images/organization.png)
+
+1. Select **People** under the organization and select **Collaborators**. In this window you can add/invite the collaborators by email for your app.
+    
+     ![](images/collaborators.png)
+
+1. Select **Teams**. Use teams to better organize people that repeatedly work together. Any collaborator within an organization can create teams. Teams allow for better management of collaborators and app permissions. For more information on creating and managing teams click [here](https://docs.microsoft.com/en-us/appcenter/dashboard/creating-and-managing-teams)
```
Comment (hsachinraj) [inline labs/vstsextend/appcenter/README.md]: drop **repeatedly**

File: labs/vstsextend/appcenter/README.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+
+    ![](images/applaunch_test.png)
+
+    For more information on **Testing Builds on Real Devices** click [here](https://docs.microsoft.com/en-us/appcenter/build/build-test-integration).
+
+## Summary
+
+As shown above, you can Automate the lifecycle of your iOS, Android, Windows, and macOS apps. Connect your repo and within minutes build in the cloud, test on thousands of real devices, distribute to beta testers and app stores. All in one place.
+
+## References
```
Comment (hsachinraj) [inline labs/vstsextend/appcenter/README.md]: I like the idea of including references to **docs** 👍

File: labs/vstsextend/appcenter/README.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+
+1. Once the build is a success, you can download the application `.apk` file and build logs from the build output.
+
+    ![](images/download_build.png)
+
+1. You can also **Distribute** the application to your collaborators or distribution groups. Users will get an email notification about your new release.
+
+   ![](images/distribute.gif)
+
+1. Since we enabled **Test on real device** feature you can view the test run performed as part of the build. Select **Test** tab. And open Test run.
```
Comment (hsachinraj) [inline labs/vstsextend/appcenter/README.md]: Testing mobile applications is extremely critical as mobile users are considered less forgiving. Statistics show that many users delete the app immediately if they find them buggy or non-performing. Users may post bad reviews in the app stores that would undermine the future product releases. Given that there are a wide range of mobile devices in the market running on various hardware and OS versions, testing the app on every device can be very expensive, more complex and challenging than a desktop or web application. Fortunately, the App Center offers thousands of physical devices to deploy apps and run tests. This is offered as a service (aka Test Cloud) making it affordable for teams of any size. You can choose from thousands of real physical devices to test the mobile app.

**Comment by sriramdasbalaji** [comment]
Hi Sachin,

I have updated the content. Please verify.

Thanks
Balaji

**Pull request merged**